src/v4.js

```diff
@@ -1,9 +1,14 @@
+import native from './native.js';
 import rng from './rng.js';
 import stringify from './stringify.js';
 
 function v4(options, buf, offset) {
   options = options || {};
 
+  if (native.randomUUID && buf == null && options.random == null && options.rng == null) {
+    return native.randomUUID();
+  }
+
```

Comment:
Maybe avoid inspecting what's inside `options` and just do this ...

```suggestion
  if (native.randomUUID && buf == null && !options) {
    return native.randomUUID();
  }

  options = options || {};

```

Reply:
Nice 👍 

Also changed `buf == null` to `!buf` in order to exactly match the `if` statement further down, and also shave of a few more bytes from the bundle ☺️